SteamList/Views/GradientView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  GradientView.swift
+//  SteamList
+//
+//  Created by Macbook on 12/01/2023.
+//
+
+import UIKit
+
+@IBDesignable
+class GradientView: UIView {
+    
+    @IBInspectable var firstColor: UIColor = UIColor.clear {
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var secondColor: UIColor = UIColor.clear {
+        didSet {
+            updateView()
+        }
+    }
+    
+    override class var layerClass: AnyClass {
+        get {
+            return CAGradientLayer.self
+        }
+    }
+    
+    func updateView() {
+        let layer = self.layer as! CAGradientLayer
```

Comment:
Not sure wether there will be any cases when layer is not an instance of `CAGradientLayer` class, but it is still good to wrap this via guard statement